akka-projection-dynamodb/src/main/resources/reference.conf

```diff
@@ -13,7 +13,8 @@ akka.projection.dynamodb {
     time-window = 5 minutes
 
     # Trying to batch insert offsets in batches of this size.
-    offset-batch-size = 20
+    # Must be less than or equal to 25 (hard limit in DynamoDB)
+    offset-batch-size = 25
```

Comment:
I would probably not have made this configurable since it's a hard limit in DynamoDB, but since we had this already in the config/settings and it wasn't used.